control/control_test.go

```diff
@@ -1300,6 +1316,7 @@ func TestPublishMetrics(t *testing.T) {
 
 		// Load plugin
 		_, err := load(c, path.Join(SnapPath, "plugin", "snap-publisher-file"))
+		So(err, ShouldBeNil)
 		<-lpe.done
 		So(err, ShouldBeNil)
```

Comment:
Previous comment applies here as well. This err check can be removed if the err will be checked before the channel returns.